resources/home/dnanexus/eggd_artemis.py

```
@@ -0,0 +1,438 @@
+    get_multiqc_report,
+    make_url,
+    find_snv_files,
+)
+from utils.util_functions import (
+    get_excluded_intervals,
+    remove_unnecessary_outputs,
+)
+
+
+def generate_sample_outputs(
+    sample,
```

Ruff (0.8.0)

Missing type annotation for function argument `sample`

(ANN001)